pkgs/development/python-modules/orbax-checkpoint/default.nix

```diff
@@ -1,43 +1,43 @@
 { lib
-, absl-py
 , buildPythonPackage
-, cached-property
-, etils
+, pythonOlder
 , fetchPypi
 , flit-core
+, absl-py
+, cached-property
+, etils
 , importlib-resources
 , jax
 , jaxlib
 , msgpack
 , nest-asyncio
 , numpy
 , protobuf
-, pytest-xdist
-, pytestCheckHook
-, pythonOlder
 , pyyaml
 , tensorstore
 , typing-extensions
+, pytest-xdist
+, pytestCheckHook
 }:
 
 buildPythonPackage rec {
   pname = "orbax-checkpoint";
-  version = "0.5.5";
+  version = "0.5.7";
   pyproject = true;
 
   disabled = pythonOlder "3.9";
 
   src = fetchPypi {
     pname = "orbax_checkpoint";
     inherit version;
-    hash = "sha256-zry5byLxFzah+e52x4yIi6roU3Jox/9mY62cujB2xlU=";
+    hash = "sha256-3hRUm4mSIKT0RUU5Z8GsLXFluBUlM0JYd0YAXwOpgTs=";
   };
 
-  nativeBuildInputs = [
+  build-system = [
     flit-core
   ];
 
-  propagatedBuildInputs = [
+  dependencies = [
```

Comment:
are these new? i've never seen them before -- are there docs to checkout for them?

if they aren't necessary for the upgrade, i propose that we leave the diff to a separate PR

Reply:
Yes, they are new (from ~2 months ago).

https://github.com/NixOS/nixpkgs/commit/4d0cca465476a1142c187198c9fe3fcd89aec6d1
https://github.com/NixOS/nixpkgs/commit/b9138b7c072f55dcfd7b8c0747c3c4291bd0b28e

However, I removed them from this PR to not clutter the git blame.